From ef9e9174c87bf4ad6431b788e7e83454d82c8209 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 10:56:54 +0800
Subject: [PATCH] Test case1-E for Change build option under BuildOptions

Description: Change build option under BuildOptions (Change to"GCC:RELEASE_*_*_CC_FLAGS = -D CRB_FLAG") in TestPkg/buildoptions.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/buildoptions.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/buildoptions.dsc b/TestPkg/buildoptions.dsc
index 9ca9e101ff..396f4f7dd3 100644
--- a/TestPkg/buildoptions.dsc
+++ b/TestPkg/buildoptions.dsc
@@ -1,5 +1,5 @@
 [BuildOptions]
-  GCC:RELEASE_*_*_CC_FLAGS             = -DMDEPKG_NDEBUG
+  GCC:RELEASE_*_*_CC_FLAGS              -D CRB_FLAG
   INTEL:RELEASE_*_*_CC_FLAGS           = /D MDEPKG_NDEBUG
   MSFT:RELEASE_*_*_CC_FLAGS            = /D MDEPKG_NDEBUG
 !if $(TOOL_CHAIN_TAG) != "XCODE5" && $(TOOL_CHAIN_TAG) != "CLANGPDB"
-- 
2.27.0.windows.1

